src/components/BannerSlider/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState, useCallback } from "react";
+
+interface bannerMockType {
```

Comment:
Please use PascalCase for interfaces. I will add role inside our .eslintrc.cjs
So, use like BannerMockType {...}